app/services/event_driven_services.py

```
@@ -0,0 +1,766 @@
+                if messages:
+                    await self._process_stream_messages(
+                        service_name, 
+                        config,
+                        messages[0][1]  # Get messages for our stream
+                    )
+                
+            except asyncio.TimeoutError:
+                # Normal timeout, continue
+                continue
+                
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)